extensions/azurecore/src/account-provider/azureAccountProvider2.ts

```diff
@@ -92,7 +92,8 @@ export class AzureAccountProvider implements azdata.AccountProvider {
 		const resourceIdMap = new Map<azdata.AzureResource, string>([
 			[azdata.AzureResource.ResourceManagement, this.metadata.settings.armResource.id],
 			[azdata.AzureResource.Sql, this.metadata.settings.sqlResource.id],
-			[azdata.AzureResource.OssRdbms, this.metadata.settings.ossRdbmsResource.id]
+			[azdata.AzureResource.OssRdbms, this.metadata.settings.ossRdbmsResource.id],
+			[azdata.AzureResource.AzureKeyVault, this.metadata.settings.azureKeyVaultResource.id]
```

Comment:
I believe you'll want to update https://github.com/microsoft/azuredatastudio/blob/master/extensions/azurecore/src/account-provider/azureAccountProvider.ts#L159 too

Reply:
Done.